<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="1" failed="2" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2021-02-06T22:08:20 CET" name="Default suite" finished-at="2021-02-06T22:09:17 CET" duration-ms="57145">
    <groups>
    </groups>
    <test started-at="2021-02-06T22:08:20 CET" name="Default test" finished-at="2021-02-06T22:09:17 CET" duration-ms="57145">
      <class name="testovi.TestYoutube">
        <test-method is-config="true" signature="createDriver()[pri:0, instance:testovi.TestYoutube@2f54a33d]" started-at="2021-02-06T22:08:20 CET" name="createDriver" finished-at="2021-02-06T22:08:29 CET" duration-ms="8772" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createDriver -->
        <test-method signature="testSearch()[pri:1, instance:testovi.TestYoutube@2f54a33d]" started-at="2021-02-06T22:08:29 CET" name="testSearch" finished-at="2021-02-06T22:08:58 CET" duration-ms="28947" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testSearch -->
        <test-method signature="pustiPesmuTest()[pri:2, instance:testovi.TestYoutube@2f54a33d]" started-at="2021-02-06T22:08:58 CET" name="pustiPesmuTest" finished-at="2021-02-06T22:09:07 CET" duration-ms="8929" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [//*[@id="video-title"]/yt-formatted-string] but found [https://www.youtube.com/watch?v=jEzuw6YnbUQ]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [//*[@id="video-title"]/yt-formatted-string] but found [https://www.youtube.com/watch?v=jEzuw6YnbUQ]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at testovi.TestYoutube.pustiPesmuTest(TestYoutube.java:48)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- pustiPesmuTest -->
        <test-method signature="testUnosaPesme()[pri:3, instance:testovi.TestYoutube@2f54a33d]" started-at="2021-02-06T22:09:07 CET" name="testUnosaPesme" finished-at="2021-02-06T22:09:17 CET" duration-ms="10370" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [https://www.youtube.com/results?search_query=KA+HASHISH+-+TONCI+HULJIC+%26+MADRE+BADESSA+FT.+PETAR+GRASO] but found [https://www.youtube.com/watch?v=jEzuw6YnbUQ]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [https://www.youtube.com/results?search_query=KA+HASHISH+-+TONCI+HULJIC+%26+MADRE+BADESSA+FT.+PETAR+GRASO] but found [https://www.youtube.com/watch?v=jEzuw6YnbUQ]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at testovi.TestYoutube.testUnosaPesme(TestYoutube.java:59)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testUnosaPesme -->
      </class> <!-- testovi.TestYoutube -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
